.gitignore

```diff
@@ -16,6 +16,8 @@ pom.xml.asc
 .hg/
 .idea
 beat-link-trigger.iml
-
+.github/scripts/**/*
+*.log
+.cache
```

Comment:
What tool was creating cache files here, making you want to exclude them? (I am not ruling out adding the log and cache exclusions, just trying to better understand them.)